algorithms/qaoa/knapsack/qaoa_knapsack.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        cost_estimation = es.estimate_cost(
</pre><p>I think it's better to move the definition of <code>valuate_params()</code> to under the <code>with ExecutionSession() ...</code> , or pass the execution session as a parameter. Otherwise, the binding of variable <code>es</code> is unclear..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>